librz/util/*.c: many warnings: cast from pointer to integer of different size

On ARMv6 builds:
```
../librz/util/axml.c: In function 'string_lookup':
../librz/util/axml.c:119:21: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  119 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
      |                     ^
../librz/util/axml.c:119:35: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  119 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
      |                                   ^
../librz/util/axml.c:129:21: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  129 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
      |                     ^
../librz/util/axml.c:129:35: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  129 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
      |                                   ^
../librz/util/axml.c:149:21: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  149 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut32) - n - 1) {
      |                     ^
../librz/util/axml.c:149:35: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  149 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut32) - n - 1) {
      |                                   ^
../librz/util/axml.c:160:21: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  160 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut32)) {
      |                     ^
../librz/util/axml.c:160:35: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  160 |                 if ((ut64)start > (ut64)data + data_size - sizeof(ut32)) {
      |                                   ^
../librz/util/axml.c:177:21: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  177 |                 if ((ut64)start16 > (ut64)data + data_size - sizeof(ut32) - n - 1) {
      |                     ^
../librz/util/axml.c:177:37: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  177 |                 if ((ut64)start16 > (ut64)data + data_size - sizeof(ut32) - n - 1) {
      |                                     ^
```
See also
```
../librz/util/thread.c: In function 'rz_th_set_affinity':
../librz/util/thread.c:134:38: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  134 |         if (sched_setaffinity((pid_t)(ut64)th->tid, sizeof(c), &c) != 0) {
      |       
```

--- a/librz/util/axml.c
+++ b/librz/util/axml.c
@@ -116,7 +116,7 @@ static char *string_lookup(string_pool_t *pool, const ut8 *data, ut64 data_size,
 
 	char *name = NULL;
 	if (pool->flags & FLAG_UTF8) {
-		if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
+		if ((uintptr_t)start > (uintptr_t)data + data_size - sizeof(ut16)) {
 			return NULL;
 		}
 
@@ -126,7 +126,7 @@ static char *string_lookup(string_pool_t *pool, const ut8 *data, ut64 data_size,
 			n = ((n & 0x7f) << 8) | *start++;
 		}
 
-		if ((ut64)start > (ut64)data + data_size - sizeof(ut16)) {
+		if ((uintptr_t)start > (uintptr_t)data + data_size - sizeof(ut16)) {
 			return NULL;
 		}
 
@@ -146,7 +146,7 @@ static char *string_lookup(string_pool_t *pool, const ut8 *data, ut64 data_size,
 			return name;
 		}
 
-		if ((ut64)start > (ut64)data + data_size - sizeof(ut32) - n - 1) {
+		if ((uintptr_t)start > (uintptr_t)data + data_size - sizeof(ut32) - n - 1) {
 			free(name);
 			return NULL;
 		}
@@ -157,7 +157,7 @@ static char *string_lookup(string_pool_t *pool, const ut8 *data, ut64 data_size,
 			*length = n;
 		}
 	} else {
-		if ((ut64)start > (ut64)data + data_size - sizeof(ut32)) {
+		if ((uintptr_t)start > (uintptr_t)data + data_size - sizeof(ut32)) {
 			return NULL;
 		}
 
@@ -174,7 +174,7 @@ static char *string_lookup(string_pool_t *pool, const ut8 *data, ut64 data_size,
 
 		name = calloc(n * 2 + 1, 1);
 
-		if ((ut64)start16 > (ut64)data + data_size - sizeof(ut32) - n - 1) {
+		if ((uintptr_t)start16 > (uintptr_t)data + data_size - sizeof(ut32) - n - 1) {
 			free(name);
 			return NULL;
 		}

--- a/librz/util/thread.c
+++ b/librz/util/thread.c
@@ -131,7 +131,7 @@ RZ_API bool rz_th_set_affinity(RZ_NONNULL RzThread *th, int cpuid) {
 	CPU_ZERO(&c);
 	CPU_SET(cpuid, &c);
 
-	if (sched_setaffinity((pid_t)(ut64)th->tid, sizeof(c), &c) != 0) {
+	if (sched_setaffinity((pid_t)th->tid, sizeof(c), &c) != 0) {
 		RZ_LOG_ERROR("thread: Failed to set cpu affinity\n");
 		return false;
 	}

